### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by andersonlebon**
here are what was done: 

- [ ] Open the project Bookstore CMS in `zeplin`.
- [ ] Use the information detailed in Zeplin to style the website.
- [ ] deploy the website on `heroku`.
- [ ] Correct linter errors

### CODE REVIEW

File: src/sass/_navbar.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    font-weight: bold;
+  }
+
+  ul {
+    margin: 0 20px;
+    gap: 25px;
+  }
+
+  li:first-child a {
+    color: $primary-color;
```
Comment (ZawadiSumba66) [inline src/sass/_navbar.scss]: - [ ] Kindly note that the color of this book link is dark and not blue.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi @andersonlebon 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Good job on making your CSS DRY 👏 
## Required Changes ♻️
- [ ] The instructions for cloning the project at getting started section are not for this repository. Kindly update.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — APPROVED** [review]
Hi @andersonlebon 👋 ,
### STATUS: APPROVED 👍 
Great job implementing the changes requested. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by andersonlebon** [comment]
Thanks for the review @ZawadiSumba66  I appreciated it

**Pull request merged**